NEWS.md

```diff
@@ -0,0 +1,10 @@
+## 2018-10-08 v1.0.0
+ * Added raml files
+## 2018-10-04 v1.0.0
+ * Initial module setup
+ * Added Jenkinsfile
+ * Added Dockerfile
+ * Updated README file
+ * Added boilterplate files
+## 2018-10-04 v1.0.0
+ * Repository creation
```

Comment:
I am not sure if there is a specific pattern for publishing news - but when I check mod-notes as an example, they have version first and then date - not sure if this matters though - also observed that versions start off with 0.1.0 and go all the way up based on minor and major updates

Reply:
Was also wondering if version should be a 0.1.0 initially -- not sure of best practice for this

Reply:
format changed. version set to 0.1.0